tracing-subscriber/src/filter/env/directive.rs

```diff
@@ -415,6 +415,7 @@ impl<T: Match + Ord> DirectiveSet<T> {
             .filter(move |d| d.cares_about(metadata))
     }
 
+    #[allow(clippy::op_ref)]
```

Comment:
Are these possibly an instance of https://github.com/rust-lang/rust-clippy/issues/2597? Would be good to document that on the `allow` attribute if that's the case.

Reply:
Ah, could be. I'll look into that.

Reply:
So I'm not sure if that's what was going on here but I managed to rewrite the code to not need the allow.